service/src/main/java/bio/terra/cbas/runsets/types/CbasMap.java

```diff
@@ -0,0 +1,54 @@
+package bio.terra.cbas.runsets.types;
+
+import bio.terra.cbas.model.ParameterTypeDefinition;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class CbasMap implements CbasValue {
+
+  private final Map<CbasValue, CbasValue> values;
+
+  private final ParameterTypeDefinition keyType;
+  private final ParameterTypeDefinition valueType;
```

Comment:
SonarCloud is detecting a code smell saying unused `keyType` and `valueType`. Are they really unused?